pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -1355,6 +1357,26 @@ func (r *ReconcileClusterPool) createCloudBuilder(pool *hivev1.ClusterPool, logg
 		cloudBuilder.NetworkName = platform.Ovirt.NetworkName
 
 		return cloudBuilder, nil
+	case platform.Nutanix != nil:
```

Comment:
Did you try a clusterpool? Did it work? How did you plumb in the two extra Secrets?

Reply:
With the new secretes addition it works it works